cypress/integration/styling_spec.js

```diff
@@ -4,4 +4,75 @@ describe('Styling', () => {
       .get('nav')
       .should('have.css', 'position', 'fixed')
     })
+
+    it("Make all the project images the same height", () => {
+      cy
+      .get('.project_image')
+      .should('have.css', 'height')
+      
+    })
+
+    it("Make all the project images the same width", () => {
+      cy
+      .get('.project_image')
+      .should('have.css', 'width')
+      
+    })
+
+    it("Give each project a solid black border of 2px", () => {
+      cy
+      .get('.project_card')
+      .should('have.css', 'border-style', 'solid')
+      .and('have.css', 'border-color', 'rgb(0, 0, 0)')
+      .and('have.css', 'border-width', '2px')
+    })
+
+    it("Give the project description a semi-transparent background with a color hex of #0c1427", () => {
+      cy
+      .get('.project_description')
+      .should('have.css', 'background-color', 'rgba(12, 20, 39, 0.3)')
+    })
+
+    it("Change the project description text to white", () => {
+      cy
+      .get('.project_description')
+      .should('have.css', 'color', 'rgb(255, 255, 255)')
+    })
+
+    it("Lay your project description over your project image using position absolute", () => {
+      cy
+      .get('.project_description')
+      .should('have.css', 'position', 'absolute')
+    })
+
+    it("Give you project description's parent element a relative position", () => {
+      cy
+      .get('.project_card')
+      .should('have.css', 'position', 'relative')
+    })
+
+    it("Fix your nav bar to the top using the z-index", () => {
+      cy
+      .get('nav')
+      .should('have.css', 'z-index', '100')
```

Comment:
Let me know if I'm wrong about this, but I think the `z-index` here only needs to be higher than the z-index of the elements we want it to be on top of. So in this case any z-index greater than 0 would work.

There is a way in cypress to test this:
- get the value of the element's `z-index` attribute (which will be a string), parse it to an integer, and assign it to a variable
- assert that the value of this variable is greater than 0.

Some helpful resources here:
https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Closures
There is a good code example here where we grab a property of the element, assign it to a variable, and then do `expect` assertions against it.

https://api.jquery.com/css/
This is the method you would use to get the value of an element's css attribute

https://docs.cypress.io/guides/references/assertions.html#Chai
Cypress uses the `chai` library for its `expect` syntax, which is slightly different than the `expect` library you might be familiar with from jest. The syntax for `greaterThan` is in the `BDD Assertions` table.